### DISCUSSION TITLE
Using lighthouse to check for performance issues and other improvable factors

### DISCUSSION THREAD

**Discussion by ObiKeahloa** [monkeytypegame/monkeytype#1101] [Ideas]
After the New V1.7.10 update Lighthouse results show that overall performance is better than before , maybe it was the code refactoring?

But what I wanted to say was why not use Lighthouse for every release to check for places where performance can be improved?
So that the next release has better performance?

**Comment by unrealapex** [comment]
https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fmonkeytype.com%2F
To get a report, use [web.dev](https://web.dev/measure/).

**Reply by ObiKeahloa** [reply]
Well performance is in the low 20-30s

**Comment by walter-0** [comment]
I don't know if mobile is important for a typing site, but the Core Web Vitals score can be improved by a lot on desktop.

https://pagespeed.web.dev/analysis/https-monkeytype-com/2r2mi92x03?form_factor=desktop

**Desktop**

![pagespeed web dev_analysis_https-monkeytype-com_2r2mi92x03_form_factor=desktop](https://github.com/user-attachments/assets/64f922ad-4af8-4798-9864-b7fd17d19c9e)

**Mobile**

![pagespeed web dev_analysis_https-monkeytype-com_2r2mi92x03_form_factor=mobile](https://github.com/user-attachments/assets/9466e8dd-9c97-49f5-9950-cee6761c0d5a)
